app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_line = command.split(" ")
+    file_1 = command_line[1]
+    file_2 = command_line[2]
```

Comment:
Use unpacking for this (example: `values = [1, 2]; a, b = values`)

Reply:
Add validation for command (command should be "cp" and length of command should be equal exactly to 3)